scripts/run_mypy_checks.py

```diff
@@ -61,6 +61,10 @@
     'core/domain/feedback_services_test.py',
     'core/domain/html_validation_service.py',
     'core/domain/html_validation_service_test.py',
+    'core/domain/learner_group_fetchers.py',
+    'core/domain/learner_group_fetchers_test.py',
+    'core/domain/learner_group_services.py',
+    'core/domain/learner_group_services_test.py',
```

Comment:
Do not add this here, new files should be MyPy typed.

Reply:
I was focusing on getting the feature up first, but I have added mypy tests for them.